spec/controllers/errors_controller_spec.rb

```diff
@@ -23,4 +23,15 @@
       expect(response).to have_http_status(:unprocessable_entity)
     end
   end
+
+  describe "removes flash messages" do
+    it "will remove any flash messages" do
+      get :not_found, flash: { success: "Success" }
```

Comment:
you could probably just loop through these actions and have one assertion in the block passing it each action to request since they're all doing the same assertion. It's not a blocker and i normally don't mind tests being explicit but in this scenario it may make sense